docs/example/README.md

```diff
@@ -28,65 +42,142 @@ Use the [installation guide](/docs/installation_guide.md) to install the `osm` c
 osm install
 ```
 
-## Install Applications
+## Deploying the Bookstore Demo Applications
 The `Bookstore`, `Bookbuyer`, `Bookthief`, `Bookwarehouse` demo applications will be installed in their respective Kubernetes Namespaces. In order for these applications to be injected with a Envoy sidecar automatically, we must add the Namespaces to be monitored by the mesh.
 
+### Create the Bookstore Application Namespaces
 ```bash
-kubectl create ns bookstore
-kubectl create ns bookbuyer
-kubectl create ns bookthief
-kubectl create ns bookwarehouse
-osm namespace add bookstore bookbuyer bookthief bookwarehouse
+for i in bookstore bookbuyer bookthief bookwarehouse; do kubectl create ns $i; done
 ```
-
+### Onboard the Namespaces to the OSM Mesh
+```bash
+for i in bookstore bookbuyer bookthief bookwarehouse; do osm namespace add $i; done
+```
+### Deploy the Bookstore Application
 Install `Bookstore`, `Bookbuyer`, `Bookthief`, `Bookwarehouse`.
 ```bash
-kubectl apply -f docs/example/manifests/apps/
+kubectl create -f docs/example/manifests/apps/
 ```
 
-### What Got Installed
+### Checkpoint: What Got Installed?
+The following are the key components of the demo application:
+
 - A Kubernetes Deployment, Kubernetes Service, and Kubernetes ServiceAccount for each application.
 - A *root service* called `bookstore` which other applications will use to direct traffic to the Bookbuyer application.
-- An [SMI TrafficSplit][3] resource which specifies how much traffic should to each version of `Bookstore`.
+- An [SMI TrafficSplit][3] resource which specifies how much traffic should go to each version of `Bookstore`.
+
+To view these resources on your cluster, run the following commands:
+```
+kubectl get svc
+kubectl get deploy --all-namespaces
+kubectl get trafficsplit -n bookstore
+```
+
+A simple toplogy view of the Bookstore application looks like the following:
+![Bookstore Application Topology](/img/book-thief-app-topology.jpg "Bookstore Application Topology")
 
 *Note: At the moment, you must configure a TrafficSplit object to get your applications set up correctly for inbound traffic because it helps us properly configure the dataplane. We're working on removing the need for this entirely.* [#1370](https://github.com/openservicemesh/osm/issues/1370)
 
-### View Application UIs
+### View the Application UIs
+We will now setup client port forwarding, so we can access the services in the Kubernetes cluster. It is best to start a new terminal session for running the port forwarding script to maintain the port forwarding session, while using the original terminal to continue to issue commands. The port-forward-all.sh script will look for a ".env" file for variables. We will use the reference .env.examples file and then run the port forwarding script. 
+
+In a new terminal session, run the following commands to enable port forwarding into the Kubernetes cluster.
 ```bash
+cp .env.example .env
```

Comment:
This line should reference why the `.env` is needed (demo/README.md)

Reply:
Updated more context for the .env file requirement.